doc/Type/Supply.rakudoc

```diff
@@ -15,7 +15,13 @@ It is a thread-safe implementation of the
 L<Observer Pattern|https://en.wikipedia.org/wiki/Observer_pattern>,
 and central to supporting reactive programming in Raku.
 
-There are two types of Supplies: C<live> and C<on demand>. When tapping into a
+There are two types of Supplies: C<live> and C<on demand>. A C<live>
+supply is like a TV broadcast: those who tune in don't get previously emitted
+values. An C<on-demand> broadcast is like Netflix: everyone who starts streaming
```

Comment:
I would prefer to not use Netflix specifically - would you be OK with using something generic like “A video streaming service”?